eisbuk-admin/src/components/atoms/IntervalCard/IntervalCard.tsx

```diff
@@ -64,30 +65,13 @@ const BookingCard: React.FC<Props> = ({
    * - you need to refactor this to use interval instead of calculating from duration
    * - should be straight forward, no fancy calculations or conversions (the start and end times are already received as `interval` properties as strings)
    */
-  // const startTimeISO = date.toISOTime().substring(0, 5);
-  // const endTimeISO = date
-  //   .plus({ minutes: Number(bookedDuration) - 10 })
-  //   .toISOTime()
-  //   .substring(0, 5);
-
-  // const timeSpan = (
-  //   <Box className={classes.time}>
-  //     <Typography component="h2">
-  //       <Typography color="primary" display="inline" variant="h5">
-  //         <strong>{startTimeISO}</strong>
-  //       </Typography>
-  //       <Typography className={classes.endTime} display="inline" variant="h6">
-  //         - {endTimeISO}
-  //       </Typography>
-  //     </Typography>
-  //   </Box>
-  // );
 
   const handleClick = () => {
     /**
      * @FOR_FADWA_TASK_2
      * should call proper booking operation (with respect to `booked` flag)
      */
```

Comment:
This is exactly the expected functionality, so you can remove the comment. Just a little cleaner way (since it's only one operation: you can remove the `{}` block and simply "return" `booked ? cancelBooking() : bookInterval()`. This will not return anything since both functions are `void`, just like the `handleClick` function, so type correct.